charts/dex.yaml

```diff
@@ -26,8 +26,6 @@ ingress:
   enabled: true
   annotations:
     nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
-    nginx.ingress.kubernetes.io/proxy-ssl-verify: "on"
-    nginx.ingress.kubernetes.io/proxy-ssl-secret: "dex-ca-cert"
```

Comment:
Why remove those?

Reply:
From the above review, we remove the secret `dex-ca-cert` so it should not be needed here again I think!

Reply:
Oh right that's why I left the CA cert... My bad. Let's leave it as-is for now, open a debt ticket to set this up (and test it).